src/layouts/Default.vue

```diff
@@ -2,8 +2,8 @@
     <div class="layout">
         <header class="masthead">
             <b-navbar class="container justify-content-center" toggleable="lg">
-                <b-navbar-brand href="#" class="align-middle">
-                    <g-image src="~/images/logos/logo_wo_text.png" class="d-inline-block" width="50" alt="GMN Logo"/>
+                <b-navbar-brand href="/" class="align-middle">
+                    <g-image src="~/images/logos/logo_white_wo_text.png" class="d-inline-block" width="50" alt="GMN Logo"/>
```

Comment:
Great job! Could you please replace the logo with the one with a dark background? This doesn't show well on a browser tab. You can find the logo [here](https://drive.google.com/file/d/1--GoMZ5oj0adMG5GZY5t6adL45YbtZoE/view?usp=sharing) or in the logo folder on Google drive. Thanks.

Reply:
Do you have a screenshot when it does not look good. I used the transparent background so the background color is not bothering (no need to have the same in the image as on the navbar)

Reply:
Here is a screenshot:
![logo](https://user-images.githubusercontent.com/39836912/153212575-48f254c7-5868-4ae3-8181-602610da8a0a.PNG)

Reply:
It is not totally bad, we can leave it like that.

Reply:
As ok. So really the favicon. I thought the logo on the navbar. It looks good for me because I have the black mode on Firefox. I will have a look later

Reply:
What you pointed it not the favicon in the tab but the logo in the navbar (menu)

Reply:
No, it wasn't because of that. It was because I said logo instead of favicon. :)